src/user/using-the-aggregate-data-entry-app.md

```diff
@@ -0,0 +1,283 @@
+    value range.
+
+1.  Click on "Add limits"
+
+    ![](resources/images/data_entry_beta/add_limits.png)
+
+3.  Enter **Min limit** and **Max limit**.
+
+4.  Click **Save**.
+
+    ![](resources/images/data_entry_beta/save_new_limits.png)
+    
+    If values don't fall within the new value range the next time you enter
+    data, the data entry cell will appear with a red background.
+
+## Display a data value's history { #aggregate_data_entry_app.display_data_value_history } 
+
+You can display the last 12 values registered for a field.
+
+1.  Open the **Data Entry** app (beta).
+
+1.  Open an existing data entry form.
+
+1.  Open the data value's details
+
+4.  Click **History**.
+
+![](resources/images/data_entry_beta/history.png)
+
+## Display a data value's audit log { #aggregate_data_entry_app.display_data_value_audit_log } 
+
+The audit log allows you to view other data values which have been entered
+prior to the current value. The audit trail also shows when the data value was
+altered and which user who made the changes.
+
+1.  Open the **Data Entry** app (beta).
+
+1.  Open an existing data entry form.
+
+1.  Open the data value's details
+
+4.  Click **Audit log**.
+
+![](resources/images/data_entry_beta/audit_log.png)
+
+## Enter data offline { #aggregate_data_entry_app.enter_data_offline } 
+
+The **Data Entry** app (beta) works even if you don't have a stable Internet
+connection during data entry. When you don't have an internet
+connection, the data you enter is saved to your local computer. When the
+Internet connection is back, the app will push the data to the server.
+
+> **Note**
+> 
+> To use this functionality, you must login to the server while you've
+> an Internet connection.
+
+-   When you're connected to the Internet, the app displays this message
+    at the top of the data entry form:
+    
+    ![](resources/images/data_entry_beta/data_entry_online1.png)
+
+-   If your Internet connection breaks during data entry, the app
+    detects it and displays this message:
+    
+    ![](resources/images/data_entry_beta/data_entry_offline1.png)
+    
+    Now your data will be stored locally. You can continue to enter data
+    as normal.
+
+-   Once you have entered data and the app detects that the Internet connection
+    is back, the data will be sent automatically:
+
+## Multi-organisation unit data entry { #aggregate_data_entry_app.data_entry_multiple_organisation_units } 
+
+The **Data Entry** app (beta) currently does not support multi-organisation
+units!
+
+## See also { #aggregate_data_entry_app.data_entry_app_see_also } 
+
+  - [Control data
+    quality](https://docs.dhis2.org/master/en/user/html/control_data_quality.html)
+
+  - [Manage data sets and data entry
+    forms](https://docs.dhis2.org/master/en/user/html/manage_data_set.html)
+
+  - [Using the Maintenance
+    app](https://docs.dhis2.org/master/en/user/html/maintenance_app.html)
```

Comment:
Refer to unique anchors directly rather than full URLs which may change.
(see https://docs.dhis2.org/en/implement/support-and-documentation/dhis2-documentation-guide.html#name_of_section)

```suggestion
  - [Manage data sets and data entry
    forms](#manage_data_set)
    
  - [Control data
    quality](#control_data_quality)
```